implementations/rust/ockam/ockam_api/src/nodes/registry.rs

```diff
@@ -1,10 +1,37 @@
 use crate::nodes::service::Alias;
 use ockam_core::compat::collections::BTreeMap;
-use ockam_core::Address;
+use ockam_core::{Address, Result, Route};
+use ockam_identity::Identity;
+use ockam_vault::Vault;
 
 #[derive(Default)]
 pub(crate) struct SecureChannelInfo {}
 
+pub(crate) struct OrchestratorSecureChannelInfo {
+    addr: Address,
+}
+
+impl OrchestratorSecureChannelInfo {
+    pub(crate) fn new(addr: Address) -> Self {
+        Self { addr }
+    }
+
+    pub(crate) fn addr(&self) -> &Address {
+        &self.addr
+    }
+}
+
+#[derive(Default, Hash, Ord, PartialOrd, Eq, PartialEq)]
+pub(crate) struct IdentityRouteKey(Vec<u8>);
+
+impl IdentityRouteKey {
+    pub(crate) async fn new(identity: &Identity<Vault>, route: &Route) -> Result<Self> {
+        let mut key = identity.export().await?;
```

Comment:
Could you please elaborate why identity should be present in the key? NodeManager can't have more than one

Reply:
Not needed! This is a remnant of a wrong approach I did when I started to work on this PR. I'll leave just the `route` as a key. The idea is to reuse the secure channels that are used between node and orchestrator (i.e. controller + project nodes).

Reply:
I was now reviewing the open issues we have and it seems we want to support multiple identities on a node: https://github.com/build-trust/ockam/issues/3186

So I guess I should revert and use `IdentityRouteKey` as a key.